common/src/main/java/com/wynntils/gui/screens/maps/MainMapScreen.java

```diff
@@ -340,7 +345,7 @@ public boolean mouseClicked(double mouseX, double mouseY, int button) {
                 return true;
             }
 
-            if (hovered != null) {
+            if (hovered != null && !(hovered instanceof TerritoryPoi)) {
```

Comment:
Maybe move to onClick code to the hovered poi itself?

Reply:
We could eventually. Didn't want to refactor Pois even more.

Reply:
I dont see that many use cases though

Reply:
Seaskipper map will have custom behavior on hover click, but yeah.